lua-http-parser.c

```diff
@@ -213,7 +249,12 @@ static int lhp_push_nil_event(lhttp_parser* lparser, int cb_id) {
     if ( ! lua_checkstack(L, 5) ) return -1;
 
     lua_rawgeti(L, ST_FENV_IDX, cb_id);
-    lua_pushnil(L);
+    if(cb_id == CB_ON_CHUNK_HEADER){
```

Comment:
Purely stylistic - please put the constant on the left-hand-side:

```
if ( CB_ON_CHUNK_HEADER == cb_id ) {
```

If you get into this habit, then this will fail if the "==" is a "=".